Packs/DemistoRESTAPI/Integrations/DemistoRESTAPI/README.md

```diff
@@ -14,6 +14,7 @@ Use Demisto REST APIs
     | Trust any certificate (not secure) | Trust any certificate \(not secure\). | False        |
     | Demisto Server API Key |  | False        |
     | Use system proxy settings | Use system proxy settings. | False        |
+    | Use tenant | Whether API calls should be made to the current tenant instead of the master tenant. | False |
```

Comment:
hello @dansterenson I believe you mean `tenant host` and not master tenant? This is confusing in MT environments because the _main_ can be on another host than the tenant.